resources/standards-data/ldml-keyboards/techpreview/fetch-latest-cldr-techpreview.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# Copyright:    © SIL International.
+# Description:  Import new files from CLDR tech preview
+# Create Date:  17 Oct 2022
+# Authors:      Steven R. Loomis (SRL)
+
+set -eu
+
+if [[ $# -ne 1 ]];
+then
+    echo >&2 "Usage: $0 <cldr-dir>"
+    exit 1
+fi
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/build/jq.inc.sh"
+
+cd "$THIS_SCRIPT_PATH"
+
+CLDR_DIR="$1"
+shift
+
+KEYBOARDS_DIR="${CLDR_DIR}/keyboards"
+DTD_DIR="${KEYBOARDS_DIR}/dtd"
+IMPORT_DIR="${KEYBOARDS_DIR}/import"
+DATA_DIR="${KEYBOARDS_DIR}/3.0"
+TEST_DIR="${KEYBOARDS_DIR}/test"
+
+# a file to check
+CHECK_1="${DTD_DIR}/ldmlKeyboard.dtd"      # Critical, present in prior CLDR
+CHECK_2="${DTD_DIR}/ldmlKeyboardTest.dtd"  # Only in Keyboard 3.0+
+
+if [[ ! -f "${CHECK_1}" ]];
+then
+    die "$0: error: ${CHECK_1} did not exist: is ${CLDR_DIR} a valid CLDR keyboard directory?"
```

Comment:
```suggestion
    die "${CHECK_1} did not exist: is ${CLDR_DIR} a valid CLDR keyboard directory?"
```

We'll improve `die` to put the script name and "error" text on each line